server/controllers/userFollow.js

```diff
@@ -0,0 +1,161 @@
+import isEmpty from "lodash.isempty";
+import models from "@models";
+import { serverError, httpResponse } from "@helpers/http";
+
+const { Followers, Users } = models;
+
+/**
+ * @description Follow a user
+ *
+ * @param {object} req HTTP request object
+ * @param {object} res HTTP response object
+ * @returns {object} HTTP response
+ */
+export const followUser = async (req, res) => {
+  const { userId } = req.user;
+  const authorId = req.userDetails.id;
+  try {
+    if (userId === authorId) {
+      return httpResponse(res, {
+        statusCode: 403,
+        success: false,
+        message: "You cannot follow yourself"
+      });
+    }
+    const follower = await Followers.findOne({
+      where: { authorId, followerId: userId }
+    });
+    if (isEmpty(follower)) {
```

Comment:
You can skip isEmpty here and simple test follower. As its value will be null if it did not match any column